### DISCUSSION TITLE
Is this container based on the official NordVPN client?

### DISCUSSION THREAD

**Discussion by Ansem93** [tmknight/docker-nordvpn#103] [General]
Hi,
just as title: is this container based on the on official client or is it based on something else like a general wireguard client?

**Comment by tmknight** [comment]
First, thank you for posting this in discussion vs opening an unnecessary issue - bravo.

Yes, this leverages the latest native client (the one you would install).

**Reply by Ansem93** [reply]
Sorry, I thought discussion and issues were actually different things and I took that section for general questions :(

I was looking for a docker with the official client since the many other I saw were just wireguard/openvpn clients. That's why I was still in doubt :( tomorrow I'll try it!

**Comment by tmknight** [comment]
You misunderstand me - I literally mean good job in not doing that.  No need for frowny face, you should have a smiley face :+1:
